website/scripts/parse-mdx.js

```diff
@@ -15,11 +15,16 @@
  *
  */
 
+
 module.exports = async function parseMdx({ node, getNode, actions }) {
+
   if (node.internal.type !== 'Mdx') return;
 
   // Mdx nodes parents are a File
-  const File = getNode(node.parent);
+  const File = await getNode(node.parent);
```

Comment:
if getNode was async before and not used as an await, this looks like a serious issue! where is/was it defined? I'm having trouble with the cross reference

Reply:
I tried without await and it worked, maybe when I edited this file it doesn't generated my components so I thought that getNode was async but was my mistake, fixing it